### PR TITLE
fixed checkstyle mistakes

### PR DESCRIPTION
**Pull request by lavrivskyi**

### CODE REVIEW

File: src/main/java/core/basesyntax/HelloWorld.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String helloMates;
+    private int variable;
```
Comment (Resci) [inline src/main/java/core/basesyntax/HelloWorld.java:9]: What does `variable` mean?

File: src/main/java/core/basesyntax/HelloWorld.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    private int age;
+    private int size;
+
+    public static void main(String[] args) {
+        Dog dog = new Dog();
+        Cat cat = new Cat();
+    }
+
+    private String getFromOneToTen() {
+        StringBuilder result = new StringBuilder();
```
Comment (Resci) [inline src/main/java/core/basesyntax/HelloWorld.java:19]: Good point!

**Review by Resci — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/HelloWorld.java (hunk 1 above)
Comment (lavrivskyi) [reply src/main/java/core/basesyntax/HelloWorld.java]: Changed to "currentYear"

File: src/main/java/core/basesyntax/HelloWorld.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String helloMates;
```
Comment (Resci) [inline src/main/java/core/basesyntax/HelloWorld.java:8]:
```suggestion
    private String greeting;
```

**Review by Resci — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/HelloWorld.java (hunk 3 above)
Comment (lavrivskyi) [reply src/main/java/core/basesyntax/HelloWorld.java]: Fixed.

**Review by Resci — APPROVED** [review]

**Pull request closed without merging**